packages/appstore/src/components/modals/real-wallets-upgrade/components/wallets-upgrade-step-two/wallets-upgrade-step-two.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { Button, Checkbox, Icon, Modal, Text } from '@deriv/components';
+import { Localize, localize } from '@deriv/translations';
+import { observer, useStore } from '@deriv/stores';
+import WalletsImage from 'Assets/svgs/wallets';
+import './wallets-upgrade-step-two.scss';
+
+type TWalletsUpgradeStepTwoContent = {
+    value: boolean;
+    toggleCheckbox: VoidFunction;
+};
+
+type TWalletsUpgradeStepTwoFooter = {
+    handleBack: VoidFunction;
+    is_disabled: boolean;
+    upgradeToWallets: (value: boolean) => void;
+};
+
+const WalletsUpgradeStepTwoContent = observer(({ value, toggleCheckbox }: TWalletsUpgradeStepTwoContent) => {
+    const { ui } = useStore();
+    const { is_mobile } = ui;
+
+    return (
+        <div className='wallets-upgrade-step-two__content'>
+            <WalletsImage
+                className='wallets-upgrade-step-two__image'
+                image={`wallets_upgrade_step_two_${is_mobile ? 'mobile' : 'desktop'}`}
```

Comment:
The upcoming features of Trader's hub is supporting tablet view. Just wondering if the image will be fine in that tablet aspect ratio.

Reply:
The current design is not the final one, we'll account for the tablet once the new one (with a video) is being implemented